index.js

```diff
@@ -0,0 +1,52 @@
+const localtunnel = require('localtunnel')
+const express = require('express')
+const request = require('request')
+const app = express()
+const clientId = '232154912464.253884891829'
+const clientSecret = '9a4266c986fd9a04763c825c10331d4e'
```

Comment:
Please refresh these credentials. You don't want to check them in to GitHub. Anyone can become your Bot now. 😢